js/packages/eyes-cypress/src/browser/commands.js

```diff
@@ -8,9 +8,14 @@ const {eyesOpenMapValues} = require('./eyesOpenMapping');
 const {eyesCheckMapValues} = require('./eyesCheckMapping');
 const {TestResultsSummary} = require('@applitools/eyes-api');
 const refer = new Refer(value => {
-  if (!value || !value.constructor || !value.constructor.name) return false;
-  const name = value.constructor.name;
-  return name === 'HTMLDocument' || name === 'Window' || value.ownerDocument;
+  if (
+    value &&
+    (value.nodeType === Node.ELEMENT_NODE ||
+      value.nodeType === Node.DOCUMENT_NODE ||
+      (value.constructor && value.constructor.name === 'Window'))
+  ) {
+    return true;
+  }
```

Comment:
Maybe something like this? Without an `if` statement

```suggestion
  return (
    value &&
    (value.nodeType === Node.ELEMENT_NODE ||
      value.nodeType === Node.DOCUMENT_NODE ||
      (value.constructor && value.constructor.name === 'Window'))
  )
```